src/components/Posts/ComposePost.tsx

```diff
@@ -52,6 +53,9 @@ const ComposePost = ({ placeholder }: ComposePostProps) => {
           displayName: user.displayName,
         })
       );
+      if (onClose) {
+        onClose();
+      }
```

Comment:
If we make the prop optional you're right that we need an if check. However, we can simplify it using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). This basically checks to see if the function is defined before running it, so it achieves the same thing as your if statement
```suggestion
      onClose?.()
```

Reply:
Done done done